verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -75,6 +76,8 @@ interface TableRow {
     sex: string;
     age: number;
     ethnicity: string;
+    // Represents a list in a string e.g. Afghan, Albanian
```

Comment:
comment that this is comma+space separated please

Reply:
Done.